server/bin/www

```diff
@@ -2,8 +2,12 @@
 const http = require('http');
 const app = require('../app'); // The express app we just created
 
-const port = parseInt(process.env.PORT, 10) || 8000;
+const port = parseInt(process.env.PORT, 10) || 9000;
```

Comment:
I see you're using an environment variable here. You may want to include an env.sample file to describe how the file should look.